lib/src/chart/base/axis_chart/axis_chart_data.dart

```diff
@@ -936,6 +942,21 @@ class HorizontalRangeAnnotation with EquatableMixin {
   /// If none is provided, it draws with a white color.
   final Gradient? gradient;
 
+  /// Text for annotation
+  final String? text;
+
+  /// Text style for annotation
+  final TextStyle style;
+
+  /// Text rotation in degrees
+  final double? rotation;
+
+  /// Text text padding
+  final double horizontalPadding;
+
+  /// Vertical text padding
+  final double verticalPadding;
+
   /// Lerps a [HorizontalRangeAnnotation] based on [t] value, check [Tween.lerp].
   static HorizontalRangeAnnotation lerp(
```

Comment:
And please don't forget to add the new properties in the lerp() function as well.
It helps us to have transition animation for these added properties!